src/main/java/com/solvd/meta/carina/gui/utils/TakeScreenshot.java

```diff
@@ -0,0 +1,32 @@
+package com.solvd.meta.carina.gui.utils;
+
+import com.qaprosoft.carina.core.foundation.webdriver.IDriverPool;
+import org.apache.commons.io.FilenameUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+import static com.zebrunner.agent.core.registrar.TestRunRegistrar.LOGGER;
+
+public class TakeScreenshot implements IDriverPool {
+
+    public TakeScreenshot() {
+    }
+
+    public void makeScreenshot(String path) {
```

Comment:
this method saves image without extension, please compare with my example what is wrong

Reply:
1. let's store path in testdata
2. i want to have 2 methods: public void makeScreenshot(String fileName) where you manually specify filename and public void makeScreenshot() where the filename is generated automatically (by time, testname or somthing like that)

Reply:
since your method argument is fileName please rename path -> fileName